include/infinispan/hotrod/Configuration.h

```diff
@@ -54,6 +54,27 @@ class Configuration
                 socketTimeout(_socketTimeout), sslConfiguration(_sslConfiguration),tcpNoDelay(_tcpNoDelay),
                 valueSizeEstimate(_valueSizeEstimate), maxRetries(_maxRetries), balancingStrategyProducer(bsp) {}
 
+    Configuration(const std::string &_protocolVersion,
+            const ConnectionPoolConfiguration& _connectionPoolConfiguration,
+            int _connectionTimeout,
+            bool _forceReturnValue,
+            int _keySizeEstimate,
+            std::vector<ServerConfiguration> _serversConfiguration,
+            std::vector<ServerConfiguration> _failoverServersConfiguration,
```

Comment:
I prefer the way the Java client does this, via the ability to specify alternate "named clusters". This means that the switching operation would take a name parameter